core/src/modules/configLoaders/group.ts

```diff
@@ -49,19 +49,19 @@ export async function loadGroup(
   if (configObject.rules) {
     const rules = [...configObject.rules];
     const calculatesWithDate = ["lte", "gt", "lt", "gte", "eq", "ne"];
+
     for (const i in rules) {
       if (rules[i]["propertyId"]) {
         const property = await Property.findById(rules[i]["propertyId"]);
         delete rules[i]["propertyId"];
         rules[i].key = property.key;
-      }
 
-      //parses to epoch time if calculated date rule
-      if (
-        calculatesWithDate.indexOf(rules[i]["operation"]["op"]) >= 0 &&
-        rules[i]["type"] === "date"
-      ) {
-        rules[i]["match"] = Date.parse(rules[i]["match"].toString());
+        // if calculating based on a date, parse to unix timestamp
+        if (calculatesWithDate.indexOf(rules[i]["operation"]["op"]) >= 0) {
```

Comment:
You can use [`calculatesWithDate.includes()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes) here!  I too am trying to rid myself of of the `indexOf` pattern of yesterday.